styles/components/_modal.scss

```diff
@@ -39,4 +39,45 @@
   }
   &__button {
   }
+
+  &__win-heading {
+    font-size: 2.5rem;
```

Comment:
We need a font-family also.

Reply:
I will add Roboto 700 and 400 in next batch.